### PR TITLE
Simplify code

### PR DESCRIPTION
**Pull request by gaohanghang**

### CODE REVIEW

File: spring-boot-project/spring-boot/src/main/java/org/springframework/boot/web/embedded/jetty/JettyReactiveWebServerFactory.java [hunk 1]
```diff
@@ -167,7 +167,7 @@ protected JettyResourceFactory getResourceFactory() {
 	}
 
 	protected Server createJettyServer(JettyHttpHandlerAdapter servlet) {
-		int port = (getPort() >= 0) ? getPort() : 0;
+		int port = Math.max(getPort(), 0);
```
Comment (snicoll) [inline spring-boot-project/spring-boot/src/main/java/org/springframework/boot/web/embedded/jetty/JettyReactiveWebServerFactory.java]: I think the intention of the current code is clearer. We want to set the port to `0` if it's null or negative. My brain has to parse the `.max` so I am not keen to merge this change.

Comment (wilkinsona) [reply spring-boot-project/spring-boot/src/main/java/org/springframework/boot/web/embedded/jetty/JettyReactiveWebServerFactory.java]: I prefer the proposed change. I find it no harder to read than what we currently have, with the advantage that `getPort()` is only called once so it doesn't make me pause and wonder if the value could change between calls.

Comment (philwebb) [reply spring-boot-project/spring-boot/src/main/java/org/springframework/boot/web/embedded/jetty/JettyReactiveWebServerFactory.java]: I'm happy with the change as well

Comment (snicoll) [reply spring-boot-project/spring-boot/src/main/java/org/springframework/boot/web/embedded/jetty/JettyReactiveWebServerFactory.java]: Thanks both. I am not feeling strongly against it so I'll go ahead and process with the merge.

**Comment by snicoll** [comment]
@gaohanghang thank you for making your first contribution to Spring Boot.

**Comment by gaohanghang** [comment]
> I've seen you closed #18166. There is no need to create another PR when changes are requested, just push additional commits on your branch and this will update the PR.

👌, Thanks

**Pull request closed without merging**